documentation/howto/import_FH_app.adoc

```diff
@@ -0,0 +1,189 @@
+= Import Your FeedHenry Application into the IDE
+:page-layout: howto
+:page-tab: docs    
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+Use Red Hat JBoss Developer Studio’s `Import` wizard to import a FeedHenry application into your workspace. When imported, edit the application as required and push it back to the FeedHenry server. 
```

Comment:
All other howto's talks about the related plugin (i.e. in this case Mobile Hybrid Tools), not mentioning the mounthful of "Red Hat JBoss Developer Studio".